### ISSUE TITLE
Pre-trained models

### ISSUE THREAD

**Issue by annkupriyanova** [facebookresearch/fastText#479]
Hello! Which python version did you use when trained your pre-trained models?
I get an error while loading it in python 3.6.2

**Comment by EdouardGrave** [comment]
Hi @annkupriyanova,

Thank you for reporting this issue.

Did you have this problem when trying to load a binary model (with `.bin` extension) or a model in text format (with `.vec` extension)? What error did you get?

Thank you for your help!

Best,
Edouard

**Comment by annkupriyanova** [comment]
I have a problem with loading binary model through gensim.models.FastText.load()

![err](https://user-images.githubusercontent.com/19915919/38663063-756df994-3e3e-11e8-8247-36e552180524.jpeg)

**Comment by eselkin** [comment]
We had this issue too: Solved with:

from gensim.models.fasttext import FastText 
ft_vectors = FastText.load_fasttext_format(FASTTEXT_BIN_FILENAME_WITHOUT_BIN_EXTENSION)

**Comment by annkupriyanova** [comment]
Yes, I know this works. But this method _load_fasttext_format()_ doesn't allow future retraining of the model. And it's my primary task.

**Comment by eselkin** [comment]
I think if you want to use the load() function you have to use the .vec extension format.

**Comment by zhouchichun** [comment]
Hi. Do you guys have problems with memory volume problems when loading the pre-trained  model.
The pre-trained model.bin  has size of 9G, however, my memory is only 4G.  Can anyone give me a clue to solve this problem. Thanks so much!
